src/components/button/README.md

```diff
@@ -44,16 +41,10 @@ const myButton: Button = (
 
 ### Icon button
 
-Please note `type={ButtonTypes.default}` is optional (can be omitted):
+Please note `"default"` is optional (can be omitted):
 
 ```typescript
 const myButton: Button = (
-  <Button
-    disabled={true}
-    type={ButtonTypes.default}
-    icon={"plus"}
-    onClick={this.myHandler}
-    label="Press me"
-  />
+  <Button disabled={true} type="default" icon={"plus"} onClick={this.myHandler} label="Press me" />
```

Comment:
I suggest to use icon the same way as in <TextInput>, thus passing the component, not a string.